src/Swarm/TUI/View.hs

```diff
@@ -627,6 +634,10 @@ robotsListWidget s = hCenter table
     , "Position"
     , "Inventory"
     , "Status"
+    , "Actions"
+    , "Commands"
+    , "Cycles"
+    , "Activity"
```

Comment:
This robots modal is getting very wide.  When I created a robot which happened to have a sort-of-long auto-generated name, it became so wide that the `Log` column was cut off.  Perhaps we can use abbreviations for some of these headers?  The contents of the columns is not very wide, it's just the headers.

Maybe something like  `Pos Inv Status Acts Cmds Cycs Activity Log` ?